docs/proposals/003-endpoint-picker-protocol/README.md

```diff
@@ -12,7 +12,7 @@ The EPP MUST implement the Envoy
 [external processing service](https://www.envoyproxy.io/docs/envoy/latest/api-v3/service/ext_proc/v3/external_processor)protocol.
 
 For each HTTP request, the EPP MUST communicate to the proxy the picked model server endpoint, via
-adding the `target-pod` HTTP header in the request, or otherwise return an error.
+adding the `x-gateway-destination-endpoint` HTTP header in the request and as an unstructured entry in the [dynamic_metadata](https://github.com/envoyproxy/go-control-plane/blob/c19bf63a811c90bf9e02f8e0dc1dcef94931ebb4/envoy/service/ext_proc/v3/external_processor.pb.go#L320) field of the ext-proc response, or otherwise return an error.
```

Comment:
Just to clarify - we're setting this endpoint information here in two places to ensure a wide variety of implementations can support this? What should an implementation do if both values are set but differ? Should one have precedent over the other?

Reply:
Good point, I think we can say that the behavior is not deterministic if this happens, wdyt?

Reply:
done